src/components/landing-page/ProblemCard.tsx

```diff
@@ -4,8 +4,7 @@ import NextLink from "next/link";
 interface ProblemCardProps {
   href: string;
   new: boolean;
-  qubits: number;
-  speedup: "polynomial" | "superpolynomial";
+  tags: string[];
```

Comment:
We should maybe add documentation for the props. Not necessary to merge this PR, just in general

Reply:
now tracked in https://github.com/ProvideQ/toolbox-web/issues/32